sdk/src/clearingHouseUser.ts

```
@@ -401,28 +401,32 @@ export class ClearingHouseUser {
 				const market = this.clearingHouse.getMarketAccount(
 					marketPosition.marketIndex
 				);
-				let positionUnrealizedPnl = calculatePositionPNL(
-					market,
-					marketPosition,
-					withFunding,
-					this.getOracleDataForMarket(market.marketIndex)
-				);
+				if (market) {
```

dont like this, will lead to silent failures